flac.md

```diff
@@ -239,14 +239,15 @@ Data     | Description
 
 ## Seektable
 
-The seektable metadata block can be used to store seek points. It is possible to seek to any given sample in a FLAC stream without a seek table, but the delay can be unpredictable since the bitrate MAY vary widely within a stream. By adding seek points to a stream, this delay can be significantly reduced. Each seek point takes 18 bytes, so a seek table with 1% resolution within a stream adds less than 2KB of data. There MUST NOT be more than one seektable metadata block in a stream, but the table can have any number of seek points. There is also a special 'placeholder' seekpoint which will be ignored by decoders but which can be used to reserve space for future seek point insertion.
+The seektable metadata block can be used to store seek points. It is possible to seek to any given sample in a FLAC stream without a seek table, but the delay can be unpredictable since the bitrate may vary widely within a stream. By adding seek points to a stream, this delay can be significantly reduced.  There MUST NOT be more than one seektable metadata block in a stream, but the table can have any number of seek points.
+
+Each seek point takes 18 bytes, so a seek table with 1% resolution within a stream adds less than 2KB of data. The number of seek points is implied by the metadata header 'length' field, i.e. equal to length / 18. There is also a special 'placeholder' seekpoint which will be ignored by decoders but which can be used to reserve space for future seek point insertion.
```

Comment:
nit: I would add a space at `2 KB`

Reply:
In #167 I spelled these out (megabyte, gigabyte). Probably best to keep it consistent. As the document has a lot of mentions of bytes, perhaps best to spell this one out? What do you think?

Reply:
Yes, let’s keep it consistent.